tests/ut/codegen/test_pto_codegen_ops.py

```
@@ -1720,6 +1720,69 @@ def kernel(
+        - AutoTileMatmulL0 rewrites each matmul_acc into an inner K-loop with a
+          fresh IterArg ("l0_c" accumulator), whose MemRef base differs from the
+          outer pipeline loop's acc IterArg base.
+        MemoryReuse fails to unify these bases, so AllocateMemoryAddr assigns
+        different physical addresses (e.g. addr=0 and addr=16384).
+        YieldFixupMutator then inserts tile.move acc→acc between them, forming
+        a round-trip (0→16384→0) that ptoas rejects on Ascend 910B.
+        Codegen must elide all acc→acc tile.move ops unconditionally (#1352).
+
+        Dimensions: K_tile=128, N=256 on 910B.  K_tile=128 exceeds the
+        effective L0B capacity for N=256 (L0B=64KB double-buffered → 32KB;
+        128×256×2=32KB > 32KB-1), so ChooseL0Tile picks K_L0=64, creating the
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)